qemu/tests/cfg/blockdev_commit.cfg

```diff
@@ -33,6 +33,7 @@
     #mode = "absolute-paths"
     base_tag = sn1
     top_tag = sn3
+    rebase_mode = unsafe
```

Comment:
There are two different modes in which "rebase" can operate: safe mode and unsafe mode, if  rebase_mode is defined in global scope in cfg files, 'unsafe' will always be used for rebase operation, so, why not using variants here, which can let rebase choose safe mode or unsafe mode?

Reply:
In all these test cases, only unsafe mode is needed for qemu-img rebase.